docs/api/SystemAudioPlayerPlugin.md

```diff
@@ -96,6 +96,7 @@ SystemAudioPlayer interface methods:
 | [playbuffer](#playbuffer) | Buffers the audio playback on the specified player |
 | [resume](#resume) | Resumes playback on the specified player |
 | [setMixerLevels](#setMixerLevels) | Sets the audio level on the specified player |
+| [setSmartVolControl](#setSmartVolControl) | Sets the smart audio volume level on the specified player |
```

Comment:
These docs are autogenerated. These should not be manually edited. You would need to change the SystemAudioPlayer.json file and add the API there.

Reply:
Hi Anand, As you suggest , updated the new api information added as part of SystemAudioPlayer.json